jobs/import/templates/bin/run

```diff
@@ -1,12 +1,28 @@
 #!/bin/bash
 set -eu
 export HOME=/home/vcap
-sed -e '
+bin=/var/vcap/packages/shield/bin
+export SHIELD_CORE=<%= p("core") %>
+echo "set api to <%= p("domain") %>"
+$bin/shield api -k https://<%= p("domain") %> <%= p("core") %>
+echo "login to shield"
+$bin/shield login -u <%= p("username") %> -p <%= p("password") %>
+#TODO: create and remove token or keep token.
+echo "creating token"
+token=$($bin/shield create-auth-token tmp)
+echo "$token"
+
+echo "importing now......"
+sed -e "
```

Comment:
Is this now required?  I.e. you can no longer use a pre-generated token?

Reply:
well a pre-generated token isn't necessary anymore with this.
but if you want to give ppl an option i'm okay with that